es-design-system/pages/organisms/es-cta-card.vue

```diff
@@ -0,0 +1,455 @@
+<template>
+    <div>
+        <h1>
+            CTA Card
+        </h1>
+        <p class="mb-450">
+            A card CTA to be used in sidebars, card lists, or in a main content area. Has optional slots for
+            <code>image</code> and <code>cta</code>.
+        </p>
+        <div class="mb-450">
+            <h2 class="mb-200">
+                CTA with button, lazy image slot
+            </h2>
+            <es-cta-card
+                :heading="heading"
+                :heading-first="headingFirst"
+                narrow
+                :subtitle="subtitle">
+                <template #cta>
```

Comment:
For this example I'd recommend either making the button a link (there's an attribute you can pass to `es-button` that will do that), or make this a form i.e.

```html
<form
    method="GET"
    action="https://design.energysage.dev/">
    <es-button type="submit">
        Get quotes
    </es-button>
</form>
```